### DISCUSSION TITLE
[DESIGN] Word definitions

### DISCUSSION THREAD

**Discussion by rafaelmardojai** [dialect-app/dialect#310] [General]
We want to start working on word definitions in the near future.

This discussion servers to decide the user interface we might implement for this feature. 

### Google

![Captura desde 2023-02-13 12-49-25](https://user-images.githubusercontent.com/6210397/218534882-7f8b140b-2d37-45fa-a0b2-03e72d6f6257.png)

### DeepL
![Screenshot 2023-02-13 at 12-52-16 DeepL Translate The world's most accurate translator](https://user-images.githubusercontent.com/6210397/218534888-ef238e95-3a27-48b8-9801-6e8604ba39d7.png)

### Yandex
![Captura desde 2023-02-13 12-58-19](https://user-images.githubusercontent.com/6210397/218536207-44e59e73-ffd7-409a-9339-633352ed7fe2.png)

**Comment by rafaelmardojai** [comment]
This is a design attempt I did some time I go when we first thought about this feature:
![dialect-mockup-definitions-1](https://user-images.githubusercontent.com/6210397/218536386-10f58507-19b3-4d7e-bc5e-3269ebeebbc3.png)

**Comment by mufeedali** [comment]
Personally, I do prefer the popover style myself, i.e. Desktop 2
